SCRUM-48: Implement the NEW Map design

ValladaresBlanco opened the pull request:
Implement the new map and new taps and obstacles

ValladaresBlanco committed:
Initial commit to create a PR

ValladaresBlanco commented:
![Screenshot Jorge ](https://github.com/user-attachments/assets/42425b8f-354e-409f-99eb-3129c888baeb)


Implement new map

JoeQ30 committed:
Commit Inicial

JoeQ30 committed:
Animaciones adaptadas al multijugador

JoeQ30 committed:
Merge branch 'NEW-DEVELOP' into FIX-SCRUM-47

JoeQ30 committed:
Animaciones pulidas y mejores

JoeQ30 committed:
Se solucionó un conflicto

JoeQ30 committed:
Posible fallo arreglado

JoeQ30 committed:
Pequeña limpiza de código

Norman1410 committed:
Prueba

Norman1410 committed:
Package Network manager

Norman1410 committed:
LanLobbyState en escena

JoeQ30 committed:
Ajuste de algunas pantallas

JoeQ30 committed:
Merge branch 'FIX-SCRUM-47' of https://github.com/Norman1410/TheLastKing into FIX-SCRUM-47

JoeQ30 committed:
Ajuste de Mesh Collider

Norman1410 committed:
Cambios Mapa Demo

JoeQ30 committed:
Merge branch 'MAP-48' into FIX-SCRUM-47

JoeQ30 committed:
Ajuste de HUD

Norman1410 committed:
2da actualizacion Mapa

JoeQ30 committed:
Ajuste de un PowerUp para poder usarlo

JoeQ30 committed:
Ajustes menores

Norman1410 committed:
GameOver al caerse

Norman1410 committed:
LanLobbyState mapa

JoeQ30 committed:
Pruebas

JoeQ30 committed:
Merge branch 'FIX-SCRUM-47' into MAP-48

JoeQ30 committed:
Ajuste al player

Norman1410 committed:
Borrar basura

Norman1410 committed:
Camara Nombre Jugador

JoeQ30 committed:
Ajuste de busqueda de NetworkObject

Emanuelji1905 commented:
Agrego obstáculos en posiciones estratégicas, sin colapsar el mapa y que se vea tuanis
<img width="1094" height="710" alt="image" src="https://github.com/user-attachments/assets/2af727da-506e-4edb-8a38-3d8d76ea6563" />

Emanuelji1905 committed:
Agrego los obstáculos

Norman1410 committed:
Cambios LobbyUI

JoeQ30 committed:
Ajuste al prefab del Player

No tenía un componente que se supone que es importante, así que se le ha agregado para intentar solucionar el problema de los nombre en  multijugador

JoeQ30 committed:
Merge branch 'MAP-48' of https://github.com/Norman1410/TheLastKing into MAP-48

Norman1410 committed:
UI

Norman1410 committed:
UI Lobby 2.0

JoeQ30 committed:
Posible solución

Norman1410 committed:
Arreglo nombres

JoeQ30 committed:
Intento de arreglo

NicoMa11 committed:
Update PowerUp

Powers accommodated in the new map.

Norman1410 committed:
Update Pruebas_TheLastKing.unity

Borrar limit walls
